docs/src/starting.md

```diff
@@ -1,3 +1,6 @@
+```@meta
+DocTestFilters = r"(\d*)\.(\d{3})\d+" => s"\1.\2***"
+```
```

Comment:
In principle, it would be good to disable this filter again after the doctests in this file (see https://documenter.juliadocs.org/stable/man/doctests/#Filtering-Doctests). However, given that it is only relevant for a single dockets, I think it would be preferable to just add the filter to the relevant doctest below.